carbonmark/components/pages/Portfolio/AssetProject/index.tsx

```diff
@@ -72,6 +72,10 @@ export const AssetProject: FC<Props> = (props) => {
           }}
         />
         <CarbonmarkButton label={<Trans>Sell</Trans>} onClick={props.onSell} />
+        <CarbonmarkButton
```

Comment:
let's make sure to remove this before merging. Also, reminder when we do wire up the button, we need to pass next.js link so we can do a soft (client side) navigation, to avoid having to remount the entire app (which is causing the Login screen to reappear, which is related to this issue I've logged -> https://github.com/KlimaDAO/klimadao/issues/1174)